ovos_config/locale.py

```diff
@@ -1,5 +1,5 @@
 from dateutil.tz import gettz, tzlocal
-import ovos_config
+from ovos_config.config import Configuration
```

Comment:
why this change? this usually solves circular imports and is cleaner than importing inside a function

i think in this package we need special core and perhaps should simply import ovos_utils as utils and reference it that way, it should avoid the circular imports

Reply:
My idea here is to eventually deprecate the top-level imports so we don't have to init all the submodules when importing one of them

Reply:
Reverted these changes after more consideration since import refactoring would be a separate refactor on its own with more compat. testing to be done